knative-operator/vendor/knative.dev/eventing-operator/test/cleanup.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Do we need to vendor all these test files?

Reply:
Well, as it is also like that in the serving part of things, let us do it separately.
I don't want to do any of these 2:
- Touch serving related things in this PR (i.e. deleting vendored serving operator tests)
- Do some improvements for _only_ for eventing which doesn't exist for serving